docker-compose.yaml

```diff
@@ -2,8 +2,20 @@ version: "3.4"
 services:
   docker:
     container_name: docker-compose
-    build: .
+    build: ./dockerfile
     ports:
-      - 80:5000
+      - "5000:5000"
     restart:
       always
+
+  nginx:
+    container_name: nginx
+    build: ./dockerfile-nginx
+     volumes:
```

Comment:
indention?
+ can we use relative paths here in order to used docker-compose locally